README.md

```diff
@@ -50,8 +50,13 @@ public final class MyClassWithTracing {
   private static final Tracer tracer = Tracing.getTracer();
 
   public static void doWork() {
-    // Create a child Span of the current Span.
-    try (Scope ss = tracer.spanBuilder("MyChildWorkSpan").startScopedSpan()) {
+    // Create a child Span of the current Span. Always record events for this span and force it to 
+    // be sampled.
```

Comment:
Should the paragraph above explain why the example forces sampling?  IIUC, forcing sampling makes the example easier to try out, but it isn't the most common use case.

Reply:
Sounds good, I added some explanation here.